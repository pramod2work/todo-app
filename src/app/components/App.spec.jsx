/* global describe test expect */

import React from 'react'
import { shallow } from 'enzyme'

import AppComponent from './App'

describe('App Component', () => {
  test('Snapshot test', () => {
    const component = shallow(<AppComponent />)
    expect(component).toMatchSnapshot()
  })
})
