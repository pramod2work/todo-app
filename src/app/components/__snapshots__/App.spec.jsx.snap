// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Snapshot test 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <ForwardRef(Bootstrap(Container))
      style={
        Object {
          "marginTop": "1rem",
        }
      }
    >
      <Route
        component={[Function]}
        exact={true}
        path="/task/:taskId(add|\\\\d{0,9})"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
    </ForwardRef(Bootstrap(Container))>
  </BrowserRouter>
</Provider>
`;
